EIPS/eip-2771.md

```diff
@@ -1,147 +1,110 @@
 ---
 eip: 2771
 title: Secure Protocol for Native Meta Transactions
-author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri), Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Sachin Tomar (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman)
+description: A contract interface for receiving meta transactions through a trusted forwarder
+author: Ronan Sandford (@wighawag), Liraz Siri (@lirazsiri), Dror Tirosh (@drortirosh), Yoav Weiss (@yoavw), Alex Forshtat (@forshtat), Hadrien Croubois (@Amxx), Sachin Tomar (@tomarsachin2271), Patrick McCorry (@stonecoldpat), Nicolas Venturo (@nventuro), Fabian Vogelsteller (@frozeman), Pandapip1 (@Pandapip1)
 discussions-to: https://ethereum-magicians.org/t/erc-2771-secure-protocol-for-native-meta-transactions/4488
-status: Stagnant
+status: Review
 type: Standards Track
 category: ERC
 created: 2020-07-01
 ---
 
-## Simple Summary
-
-A contract interface for receiving meta transactions through a trusted
-forwarder.
-
 ## Abstract
 
-This ERC defines a minimal contract-level protocol that a compliant Recipient
-contract needs to support in order to be capable of accepting a meta
-transaction through a compliant Forwarder contract that it trusts to help it
-identify the address of the Transaction Signer. 
-
-No EVM-level protocol changes are proposed or required.
+This EIP defines a contract-level protocol for `Recipient` contracts to accept meta-transactions through trusted `Forwarder` contracts. No protocol changes are made. `Recipient` contracts fetch the effective `msg.sender` (`_msgSender()`) and `msg.data` (`_msgData()`) through custom getter functions. 
```

Comment:
Getter function is an implementation details, we should mention that the msg sender is encoded at the end of the calldata